sdk/storage/storage-blob/src/ContainerClient.ts

```diff
@@ -636,6 +629,28 @@ export class ContainerClient extends StorageClient {
     );
   }
 
+  /**
+   * Creates a new container under the specified account. If the container with
+   * the same name already exists, the operation fails.
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/create-container
+   *
+   * @param {ContainerCreateOptions} [options] Options to Container Create operation.
+   * @returns {Promise<Models.ContainerCreateResponse>}
+   * @memberof ContainerClient
+   */
+  public async create(
```

Comment:
Is it by design here, create container under blobserviceclient seems much nature. What's the scenario for this?

Reply:
It's track2's agreement cross language.

---
In reply to: [318951342](https://github.com/Azure/azure-sdk-for-js/pull/4874#discussion_r318951342) [](ancestors = 318951342)